### DISCUSSION TITLE
Syncing many changes broke dropdown and homepage

### DISCUSSION THREAD

**Discussion by RehmatSChawla** [alshedivat/al-folio#2570] [Q&A] [answered]
Due to issue #1913 Sync from template was not working in my repo for a few months. I fixed it recently but the PR created by the workflow was replacing a lot of my customisation in config.yml, base.scss and in some of my pages. I manually went through all the changes and did my best to preserve my settings, but once I deployed I found a huge issue:

My custom pages aren't appearing in my navbar, and my homepage - the about page - isn't available either. When I go to my website, it redirects to the "page not found" page. I can access some of the pages (Blog, Projects) via the permalinks but can't go back to the homepage from them either. And my custom pages aren't appearing in the search options either.

Before:
![image](https://github.com/user-attachments/assets/8681d249-5d90-4153-a66f-8e3898baa291)

After:
<img width="408" alt="image" src="https://github.com/user-attachments/assets/2737b88d-f1ac-4bd7-830a-e1c3cff74b2d">

I wonder if this is due to the change from html to liquid layouts and me messing up the merge because I didn't realise everything was switching to liquid. I'll continue updating about my attempts.

**Comment by george-gca** [comment, marked as answer]
Since your version (pre-liquid naming) a lot has changed. Some files were renamed, some code moved to other files and even some files changed directory. Also we added a ton of new functionalities. A lot of those renaming/moving code around was to make the code clearer and more concise. For example, the 404 page was moved to inside `pages/`, since, well, it is a page. 

Since your version is very outdated, it is better to start from scratch and apply your customization on top of the latest version. I think it will be easier for you to apply the changes you know you made than try to figure out why some changes were made and if you should or not use your code rather than the one coming from upstream. In `INSTALL.MD` we have a few tools that might help you with that.

**Reply by RehmatSChawla** [reply]
I followed your advice and am partway through re-customisation, but the dropdown option still isn't appearing in the navbar. Were any changes made to the dropdown or navbar layout that might be causing this?

**Reply by george-gca** [reply]
Most of the breaking changes were introduced in #2048. The dropdown is created in its page, but [_includes/header.liquid](https://github.com/alshedivat/al-folio/blob/master/_includes/header.liquid) is the part responsible for displaying it.

One thing that might help you is checking through the commit history to the specific file you are interested in. For this, open the file on GitHub and on the upper right part click in `History`. This will show all the changes made to this file, like [this](https://github.com/alshedivat/al-folio/commits/master/_includes/header.liquid).
